### PR TITLE
fixed default policy violation check for black duck

### PR DESCRIPTION
**Pull request by gautambaghel**

### CODE REVIEW

File: community/SI-System-and-Information-Integrity/policy-blackduck.yaml [hunk 1]
```diff
@@ -164,13 +164,13 @@ spec:
...
         object-templates:
-          - complianceType: mustnothave # Makes sure pods are not in any kind of black duck policy violations
+          - complianceType: musthave # Makes sure pods are not in any kind of black duck policy violations
             objectDefinition:
               apiVersion: v1
               kind: Pod
               metadata:
                 labels:
-                  com.blackducksoftware.com.pod.overall-status: IN_VIOLATION
+                  com.blackducksoftware.com.pod.overall-status: NOT_IN_VIOLATION
```
Comment (ycao56) [inline community/SI-System-and-Information-Integrity/policy-blackduck.yaml:173]: I want to make sure this is what you expect. By changing to `musthave` and `NOT_IN_VIOLATION`, this policy will be `NonCompliant` only when `NONE` of the pods has label `com.blackducksoftware.com.pod.overall-status=NOT_IN_VIOLATION`. This policy will be `Compliant` as long as there is `ONE` pod with label `com.blackducksoftware.com.pod.overall-status=NOT_IN_VIOLATION`

Comment (gautambaghel) [reply community/SI-System-and-Information-Integrity/policy-blackduck.yaml:173]: @ycao56 - I need all the pods to have the label `com.blackducksoftware.com.pod.overall-status=NOT_IN_VIOLATION` to be compliant.

If even 1 of the pods don't have this label then it should throw a policy violation.

^^ how can I say this?

Comment (ycao56) [reply community/SI-System-and-Information-Integrity/policy-blackduck.yaml:173]: That's not possible today. That's why the original policy used `mustnothave` and `IN_VIOLATION`. The policy will be compliant if `NONE` of the pods has `IN_VIOLATION` label.

Comment (gautambaghel) [reply community/SI-System-and-Information-Integrity/policy-blackduck.yaml:173]: @davemeurer -- it seems some stuff is not possible, not sure how it impacts the video demo

**Comment by ycao56** [comment]
close as stale.

**Pull request closed without merging**